betelgeuse/__init__.py

```diff
@@ -523,7 +440,7 @@ def create_xml_testcase(config, testcase, automation_script_format):
     '--automation-script-format',
     help=(r'The format for the automation-script field. The variables {path} '
           'and {line_number} are available and will be expanded to the test '
-          'case module path and the line number where it\'s defined '
+          "case module path and the line number where it's defined "
```

Comment:
For consistency in quotes and to not have to escape a quote, why not change it's to it is?